src/apps/ensuro/polygon/ensuro.api.jar-registry.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
you should move this file in `/common` as we want to keep the network folders as clean as possible containing only files related to positions

Reply:
Done, I moved it to `src/apps/ensuro/common/` following the same pattern as the pickle app.